README.md

```diff
@@ -1,30 +1,65 @@
 ![CMake](https://github.com/fair-acc/graph-prototype/workflows/CMake/badge.svg)
-# GNURadio 4.0 prototype
-A small proof-of-concept for evaluating efficient [directed graph](https://en.wikipedia.org/wiki/Directed_graph)-based algorithms, notably required block 
-('nodes' in graph-theory) structures, scheduling interfaces, and partial compile-time merging of
-[directed acyclic](https://en.wikipedia.org/wiki/Directed_acyclic_graph) as well as 
-[cyclic graphs](https://en.wikipedia.org/wiki/Feedback_arc_set) (aka. feedback loops).  
-
-The expressed goal is to guide the low-level API design and functionality for the upcoming
-[GNU Radio 4.0](https://github.com/gnuradio/gnuradio/tree/dev-4.0) release.
-
-A [single header version](https://raw.githubusercontent.com/fair-acc/graph-prototype/single-header/singleheader/Graph.hpp)
-is provided in the `singleheader/`subdirectory of the single-header branch.
-If you want to locally regenerate the single header file, just follow
-the [CI step](https://github.com/fair-acc/graph-prototype/blob/main/.github/workflows/single-header.yml#L38-L41).
-It can be used on [compiler-explorer](https://compiler-explorer.com/z/EG7Eb9K83) with `-std=c++23 -O3` compiler options.
-
-## Copyright & License
-Copyright (C) 2018-2024 FAIR -- Facility for Antiproton & Ion Research, Darmstadt, Germany<br/>
-Unless otherwise noted: [SPDX-License-Identifier: LGPL-3.0-or-later](https://spdx.org/licenses/LGPL-3.0-or-later.html)
-
-### Contributors
+
+# GNU Radio 4.0 prototype
+
+> [!IMPORTANT]
+> This is the GNU Radio 4.0 (GR4) prototype and is currently in a beta state. For production use, 
+> please use the GNU Radio 3.X (GR3) version found [here](https://github.com/gnuradio/gnuradio).
+> Bug reports related to this beta should be submitted [here](https://github.com/fair-acc/graph-prototype/issues), 
+> and bug reports for GNU Radio 3.X should be submitted [here](https://github.com/gnuradio/gnuradio/issues)
+
+GNU Radio is a free & open-source signal processing runtime and signal processing
+software development toolkit. Originally developed for use with software-defined
+radios and for simulating wireless communications, it's robust capabilities have
+led to adoption in hobbyist, academic, and commercial environments. GNU Radio has
+found use in software-defined radio, digital communications, nuclear physics, high-
+energy particle physics, astrophysics, radio astronomy and more!
+
+## Helpful Links
+
+* [GNU Radio Website](https://gnuradio.org)
+* [GNU Radio Wiki](https://wiki.gnuradio.org/)
+* [Github issue tracker for bug reports and feature requests](https://github.com/fair-acc/graph-prototype/issues)
+* [View the GNU Radio Mailing List Archive](https://lists.gnu.org/archive/html/discuss-gnuradio/)
+* [Subscribe to the GNU Radio Mailing List](https://lists.gnu.org/mailman/listinfo/discuss-gnuradio)
+* [GNU Radio Chatroom on Matrix](https://chat.gnuradio.org/)
```

Comment:
The link between gnuradio4 and the architecture channel seems not really obvious... I guess people in gnuradio-general do a good job in redirecting the right way but we could just directly link to the channel.
```suggestion
* [GNU Radio Chatroom on Matrix](https://chat.gnuradio.org/)
  * Specifically for discussions related to GNURadio 4.0 join the [#architecture channel](https://matrix.to/#/#architecture:gnuradio.org)
```